std_msgs/String bin_id
---
